index.js

```diff
@@ -12,19 +13,24 @@
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
 // console.log("%cQuestion 2", "color: red") 
-
+// const name = "Zach"
```

Comment:
Great! The key here was using `const` because it doesn't allow us to reassign our variable